CMakeLists.txt

```diff
@@ -1126,8 +1114,7 @@ if (LINUX)
     )
 endif()
 
-# TODO include target rather than libs to also bring in transitive dependencies
-set(libcsound_LIBS ${LIBSNDFILE_LIBRARY})
+set(libcsound_LIBS "")
```

Comment:
Is this actually populated with anything?

Reply:
Yes, several things are appended later